testing/opae-c/test_init_c.cpp

```diff
@@ -290,7 +290,9 @@ class init_ase_cfg_p : public ::testing::TestWithParam<const char*> {
       dirs_.pop();
     }
     // restore the opae_ase.cfg file at OPAE_ASE_CFG_SRC_PATH
-    rename(tmpfile, (char *)OPAE_ASE_CFG_SRC_PATH);
+    if (rename(src_cfg_file_.c_str(), (char *)OPAE_ASE_CFG_SRC_PATH)) {
+        OPAE_ERR("Rename failed. Need to manully remove the opae_ase.cfg.XXXXXX files");
```

Comment:
Probably should fail the test in this case too, or else the user may not notice..

Reply:
I added some additional logic to check the file existence before renaming. Now it should work. Sometimes it failed because of the permission issue. Should be OK.

Reply:
If this rename fails, then the user's config file is overwritten?  If so, we need to stop the test flow here.

Reply:
This is outdated code. I will add the comment to the latest code.